tests/CMakeLists.txt

```diff
@@ -1,20 +1,30 @@
+# Copyright (C) 2017 Fondazione Istituto Italiano di Tecnologia
+# Authors: Stefano Dafarra
+#          Giulio Romualdi
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later
+
 cmake_minimum_required(VERSION 3.1)
 
 set (CMAKE_CXX_STANDARD 11)
 
-
 project(UnicyclePlannerTest)
 
+# UnicycleTest
 add_executable(UnicycleTest UnicycleTest.cpp)
-
 target_link_libraries(UnicycleTest UnicyclePlanner ${iDynTree_LIBRARIES})
+add_test(NAME UnicycleTest
+  COMMAND UnicycleTest)
 
-add_executable(interpolationTest interpolationTest.cpp)
-
-target_link_libraries(interpolationTest ${iDynTree_LIBRARIES} UnicyclePlanner)
+# InterpolationTest
+add_executable(InterpolationTest interpolationTest.cpp)
+target_link_libraries(InterpolationTest ${iDynTree_LIBRARIES} UnicyclePlanner)
+add_test(NAME InterpolationTest
+  COMMAND InterpolationTest)
 
-add_test(NAME UnicycleTest
-        COMMAND UnicycleTest)
+# DcmInterpolationTest
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
In so-called "Modern CMake" (i.e. when you can require CMake >= 3, see https://steveire.wordpress.com/2017/11/05/embracing-modern-cmake/) it is better to avoid command that sets directory level properties such as `include_directory` and always use target-level commands such as `target_include_directories`.